app/controllers/phone/identity.js

```diff
@@ -1,10 +1,14 @@
 import Identity from 'ti.identity';
 
+var authPhrase = "";
+
 if (!Identity.isSupported()) {
-  alert('Biometric authentication is not supported on this device!');
+  alert('Biometric authentication is not supported on this device \nor\nno identities are enrolled!');
   $.authenticate.enabled = false;
 }
 
+Identity.AUTHENTICATION_POLICY_PASSCODE;
```

Comment:
Do we need this line? I don't think calling it like this does anything?

Reply:
You're right. I have checked and it does not affect the functionality. I will remove it in the next commit